components/brave_webtorrent/extension/manifest.json

```diff
@@ -7,7 +7,7 @@
     "scripts": ["extension/out/brave_webtorrent_background.bundle.js"],
     "persistent": true
   },
-  "permissions": ["tabs", "windows", "http://127.0.0.1:*/*"],
```

Comment:
cc @diracdeltas for security review

Reply:
seems ok since `<all_urls>` is required in the browser-laptop extension